app/controllers/api/v1/users_controller.rb

```diff
@@ -28,7 +29,20 @@ def index
 
       api :POST, '/v1/users', "Create user"
       def create
-        save_and_render_object_with_errors(@user)
+        @user.assign_attributes(user_params)
+
+        if @user.save
+          if params["user"]["organisations_users_ids"].present?
+            @user.organisations << Organisation.find_by(id: params["user"]["organisations_users_ids"])
```

Comment:
shouldn't that be run before `@user.save` ? 🤔

Reply:
nope..  `organisations_user` will be created before creating user and `<<` operator directly inserts in db